src/v/kafka/server/request_context.h

```diff
@@ -135,6 +136,10 @@ class request_context {
         return _conn->server().tx_gateway_frontend();
     }
 
+    std::unique_ptr<pandaproxy::schema_registry::api>& schema_registry() const {
```

Comment:
nit: I'm wondering why a ref to `unique_ptr` is returned, can it be a ref to SR::api instead?

Reply:
It's a `unique_ptr` because it's possible to start Redpanda without a Schema Registry, so it can be null.

Reply:
So the `api` pointer is not changed using this accessor, maybe const ref to unique_ptr can it be?